src/Rest/Routes/AbstractFormSubmit.php

```diff
@@ -403,6 +403,18 @@ protected function getIntegrationCommonSubmitOutput(array $formDetails, string $
 				$this->getFormSubmitMailer()->sendEmails($formDetails);
 			}
 
+			// Output result output items as a response key.
```

Comment:
```suggestion
			// Return result output items as a response key.
```
Sounds less confusing, pls update other instances as well is you agree